docroot/modules/humsci/hs_layouts/scss/three-column-w-image.scss

```diff
@@ -1,26 +1,32 @@
 @import 'decanter-no-markup';
 
 .three-column-w-image {
-  .image-wrapper {
-    float: left;
-    width: 50%;
-
-    @include padding(null 20px 20px null);
 
+  .image-wrapper {
+    @include padding(0 0 1em 0);
+    width: 100%;
     img {
-      max-width: 100%;
+      display: block;
+      margin: 0;
+      padding: 0;
+      width: 100%;
```

Comment:
Can we avoid the width: 100% on images. or does the container shrink if the image is smaller than 40% of the area?